script/build

```diff
@@ -72,7 +73,13 @@ if (!argv.existingBinaries) {
   prebuildLessCache()
   generateMetadata()
   generateAPIDocs()
-  binariesPromise = dumpSymbols()
+  if (!argv.generateApiDocs) {
+    binariesPromise = dumpSymbols()
+  }
+}
+
+if (argv.generateApiDocs) {
+  process.exit()
```

Comment:
I think you'll want to wait for the promise to complete before exiting, just in case.  Maybe it's better to add another conditional around the block of code below to check for the presence of that flag before adding the `.then()`s to `binariesPromise`?  That way the process will exit on its own after the promise finishes if we're only generating API docs.

Reply:
As at this stage `binariesPromise` is just a reference to `Promise.resolve()` that hasn't been called yet, there are no "in-flight" promises to wait for a resolution of as all of the above code is synchronous.

That being said, wrapping the entire block below in a check of that flag _is_ the more correct way of doing this if you want me to do that.

Reply:
Yep, using the if statement might make it a bit more clear that the promise isn't just being ignored.  `binariesPromise` gets assigned from `dumpSymbols` if `--existing-binaries` isn't passed, so there will actually be a pending promise at that point.